### PR TITLE
Fixing issue with After Create MF

### PR DESCRIPTION
**Pull request by chadevans**
Reordering to save the tag before calling the after create mf

Addresses issue #3.

### CODE REVIEW

File: src/LabelSelect/widget/LabelSelect.js [hunk 1]
```diff
@@ -168,10 +168,10 @@ define([
                         callback: function () {
                             //run the after create mf
                             if (this.aftercreatemf) {
-                                this._execMf(this._contextObj.getGuid(), this.aftercreatemf, function () {
-                                    this._contextObj.addReference(this._refAttribute, obj.getGuid());
-                                    this._saveObject();
-                                });
+                                // save the label before calling the microflow to save the new label
+                                this._contextObj.addReference(this._refAttribute, obj.getGuid());
```
Comment (ChrisdeG) [inline src/LabelSelect/widget/LabelSelect.js]: These 2 lines are in both branches. Move it outside the if(this.aftercreatemf)

**Comment by mgroeneweg** [comment]
First, thanks for helping with this! :-)
Unfortunately it still does not work. I enter a new label, as soon as I press space, it disappears from the widget. If I just ignore this situation and click save, the data is saved correctly. This only happens with labels that do not yet exist.

**Comment by chadevans** [comment]
@mgroeneweg I think your configuration is incorrect, per the documentation. You MUST have an after create mf set that commits and refreshes in the client for the auto add tag functionality to work. I have added a logging statement to help indicate this incorrect state.

**Comment by mgroeneweg** [comment]
Indeed! The test project is not complete. (I did not yet test with my own project). It did not have the after create microflow on the widget. I added existing microflow IVK_AfterCreateLabel to the after create of the widget in snippet ContextObjectSnippet. Now it works in the test project.

**Pull request merged**